pom.xml

```diff
@@ -32,35 +62,27 @@
 		<dependency>
 			<groupId>org.jline</groupId>
 			<artifactId>jline</artifactId>
```

Comment:
I guess technically the parent can get rid of *all* dependencies (ie have only dependencyManagement)